### ISSUE TITLE
Support cite collapsing

### ISSUE THREAD

**Issue by tweverett** [michel-kraemer/citeproc-java#179]
Hi,

As far as I can tell 3.0.0-beta.1 does not yet support cite collapsing as specified here:
https://docs.citationstyles.org/en/v1.0.2/specification.html#cite-collapsing

An example style that utilizes this is national-library-of-medicine.csl.

Could you please consider adding support for this?

Thank you for this awesome project!

Todd

**Comment by michel-kraemer** [comment]
You are right. This feature has not been implemented yet. It's definitely on my TODO list. Please stay tuned.

**Comment by tweverett** [comment]
Hello. I’m just checking to see if you have an ETA for this. Thank you!
